terraform/providers/aws/us_east_1_prod/prod-env.tf

```diff
@@ -0,0 +1,45 @@
+variable "ami_id" {}
+
```

Comment:
Oops...localhost, apparently (set by `application.properties`). Do we want to stand up a different prod instance in AWS for now?